vaadin-renderer-flow-parent/vaadin-renderer-flow/src/test/java/com/vaadin/flow/data/renderer/ComponentRendererTest.java

```diff
@@ -132,6 +132,44 @@ public void componentRenderer_childAttachedBeforeParent() {
 
     }
 
+    @Test
+    public void allowNullValues() {
+        UI ui = new TestUI();
+        TestUIInternals internals = (TestUIInternals) ui.getInternals();
+
+        ComponentRenderer<TestDiv, String> nullRenderer = new ComponentRenderer<>(
+                e -> null);
+
+        Element containerParent = new Element("div");
+        Element container = new Element("div");
+        KeyMapper<String> keyMapper = new KeyMapper<>();
+
+        ComponentDataGenerator<String> rendering = (ComponentDataGenerator<String>) nullRenderer
```

Comment:
When running locally, I'm getting:
```
java.lang.ClassCastException: class com.vaadin.flow.data.renderer.ComponentRenderer$1 cannot be cast to class com.vaadin.flow.data.renderer.ComponentDataGenerator (com.vaadin.flow.data.renderer.ComponentRenderer$1 and com.vaadin.flow.data.renderer.ComponentDataGenerator are in unnamed module of loader 'app')
 at com.vaadin.flow.data.renderer.ComponentRendererTest.allowNullValues([ComponentRendererTest.java:147](vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html))
```

Reply:
Fixed, this was also issue when moved to master from v23